spec.bs

```diff
@@ -37,6 +37,7 @@ urlPrefix: https://dom.spec.whatwg.org; spec: DOM
     for: AbortSignal
       text: dependent signals; url: abortsignal-dependent-signals
       text: signal abort; url:abortsignal-signal-abort
+      text: reason; url:abortsignal-abort-reason
```

Comment:
Just to double check, is this needed? I guess it is, but I'm surprised by this. It makes me wonder if we're holding AbortSignal wrong — I would expect this to be exported so other specs that use AbortSignals can propagate reasons similarly.